### PR TITLE
Dark mode

### PR DESCRIPTION
**Pull request by hawkinsji**
Main change is to support dark mode. That change revealed a requirement to make web views invisible during loading and navigation, otherwise there's a white flash between dark screens.

### CODE REVIEW

**Review by priscombebur — APPROVED** [review]

File: forAndroid/captivewebview/build.gradle [hunk 1]
```diff
@@ -23,8 +22,8 @@ android {
     }
 
     defaultConfig {
-        minSdkVersion 21
-        targetSdkVersion 28
+        minSdkVersion 26
```
Comment (neilbroadbent) [inline forAndroid/captivewebview/build.gradle:25]: Is this jump in the minSdkVersion needed?
The Android SKD min SDK version is still 19.

Comment (hawkinsji) [reply forAndroid/captivewebview/build.gradle:25]: The WebChromeClient class, at least, was updated to utilise some of the most recent interfaces, here:
https://github.com/vmware/captive-web-view/pull/8/commits/c61a34f54c9e361520c4b093e282c8e5e24b7a74

**Review by neilbroadbent — COMMENTED** [review]
Is there any version numbering for Captive Web View?
Would it be useful to have it so that we can better communicate a roadmap?

**Comment by hawkinsji** [comment]
> Is there any version numbering for Captive Web View?
> Would it be useful to have it so that we can better communicate a roadmap?

Raised an issue for that: https://github.com/vmware/captive-web-view/issues/9

**Review by lorenzo-blasa — APPROVED** [review]

**Pull request merged**